src/features/groups/components/ContainerCardList/ContainerCard/RenameContainerForm.tsx

```diff
@@ -90,7 +109,7 @@ export const RenameContainerForm = ({
 
   return (
     <Card asChild>
-      <div className="flex gap-2 items-center px-4 py-2">
+      <div className="flex gap-2 px-4 py-2">
         <div className="flex items-center justify-center bg-accent rounded-full min-w-11 h-11">
```

Comment:
Removing `items-center` is not the right approach because you want to keep the input aligned to the center.

Please utilize `self-start` class to only change the yellow icon's alignment individually.

- https://css-tricks.com/snippets/css/a-guide-to-flexbox/#aa-align-self
- https://tailwindcss.com/docs/align-self

Reply:
@nick-y-ito I see, I should avoid side-effect. Thanks!